composer.json

```diff
@@ -1,59 +1,67 @@
 {
     "name": "olcs/olcs-auth",
     "description": "OLCS Authentication",
+    "license": "MIT",
     "type": "library",
-    "repositories": [
-        {
-            "packagist": false
-        },
-        {
-            "url": "https://nexus.olcs.dev-dvsacloud.uk/repository/OLCS",
-            "type": "composer"
-        },
-        {
-            "url": "https://nexus.olcs.dev-dvsacloud.uk/repository/Packagist-Proxy",
-            "type": "composer"
-        }
-    ],
     "require": {
-        "php": ">=7.4",
-        "olcs/olcs-laminas": "~2.5.0 || ~3.0.0"
+      "php": "^7.4",
+        "laminas/laminas-mvc": "^2.7.14 || ^3.0",
+        "laminas/laminas-session": "^2.8.1",
+        "laminas/laminas-stdlib": "~2.7.9",
+        "laminas/laminas-http": "^2.5",
+        "laminas/laminas-servicemanager": "^2.7.5 || ^3.0.3",
+        "laminas/laminas-view": "^2.11",
+        "laminas/laminas-authentication": "^2.5",
+        "laminas/laminas-form": "^2.7",
+        "container-interop/container-interop": "^1.2"
```

Comment:
Could we run `--sort-packages` for Composer so that these are sorted correctly please?